src/views/shared/FooterMFA.js

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import hbs from 'handlebars-inline-precompile';
+import FooterSignout from './FooterSignout';
+
+export default FooterSignout.extend({
+  template: hbs(
+    '\
+      {{#if hasFactorPageCustomLink}}\
+        <a href="{{factorPageCustomLinkHref}}" data-se="factor-page-custom-link" \
+          class="link js-factor-page-custom-link" \
+          rel="noopener noreferrer" target="_blank">\
+          {{factorPageCustomLinkText}}\
+        </a>\
+      {{/if}}\
+      {{#if showLink}}\
+        <a href="#" class="link {{linkClassName}}" data-se="signout-link">\
```

Comment:
would it make sense to have this link string in a constant in FooterSignout and just append it here? Just wondering so you don't have to rewrite the exact same html for it here and in FooterSignout

Reply:
Implemented this with reusing FooterSignout's precompiled template function:
https://github.com/okta/okta-signin-widget/pull/1622/commits/bd59ea36ee6a9c838711cd205d1649b7c3693750